queue_services/entity-emailer/tracker/config.py

```diff
@@ -0,0 +1,113 @@
+# Copyright © 2019 Province of British Columbia
```

Comment:
did you look at using the config from the root level and just using the bind here?

Reply:
I looked at this a bit more.  I was able to figure out a way to use the root config file in the end.  It involves initializing the alembic stuff with `python manage.py db init --multidb`.  Doing that generates a 'env.py` in the migrations folder that looks at `SQLALCHEMY_BINDS`.  I did need to comment out some code in `env.py` to prevent migration script generation functionality from trying to run examine lear db.  And the migration scripts also contain more code to handle multiple databases.  It does work it seems.  I haven't put this code for now as I'm still a little hesitant there might be paths where alembic is trying to migrate/upgrade the lear db.

Reply:
cool, that sounds nice.
i wouldn't put too much more time on that. when we tackle putting this into the common services, we can finalize ... and it looks like we may leverage what you've learned here.